ember-intl/addon/helpers/format-date.ts

```diff
@@ -5,7 +5,19 @@ import { isEmpty } from '@ember/utils';
 
 import type IntlService from '../services/intl';
 
-export default class FormatDateHelper extends Helper {
+type Params = Parameters<IntlService['formatDate']>;
+type Value = Params[0];
+type Options = Params[1];
+
+interface FormatDateSignature {
+  Args: {
+    Named?: Options & { allowEmpty?: boolean };
+    Positional: [Value?, Options?];
+  };
+  Return: string;
+}
```

Comment:
I based the initial run on:

- https://github.com/ember-intl/ember-intl/blob/v6.2.2/ember-intl/addon/helpers/-format-base.d.ts#L5-L14
- https://github.com/ember-intl/ember-intl/blob/v6.2.2/ember-intl/addon/helpers/format-date.d.ts#L4